docs/advanced-usage/nesting-aggregate-metrics.md

```diff
@@ -0,0 +1,118 @@
+---
+icon: kolena/nest-16
+---
+
+# :kolena-nest-20: Nesting Aggregate Metrics
```

Comment:
Nit: does using `Composite` vs `Nesting` make a difference?

Reply:
`kolena.workflow.Composite` is a separate (and unrelated) concept — I'd prefer to use separate terminology for this feature. Do you have any suggestions over `Nesting`? Definitely room for improvement.

Reply:
Trying to find a word that can be related to all 3 highlighted use-cases, multiclass (slice/stratification), ensembles (collection), and pipeline (sequences/trees/chains) and came up short.

`compound`, `combined`, `consolidated`? If none looks like an improvement, `Nesting` it is.

Reply:
Went with `Nested Test Case Metrics` per discussion on Slack 👍